web/src/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { Provider } from 'react-redux'
+
+import { store } from './ts/store'
```

Comment:
I moved my files from src to src_old
So, you should miss /ts/ in your path (remove the folder, and just use './store'